addon/components/columns/draggable.js

```diff
@@ -0,0 +1,77 @@
+import Ember from 'ember';
+import Base from 'ember-light-table/components/columns/base';
+import layout from 'ember-light-table/templates/components/columns/base';
+
+const {
+  computed
+} = Ember;
+
+/**
+ * @module Column Types
+ * @class Draggable Column
+ */
+const draggable = Base.extend({
+  layout,
+  attributeBindings: ['width', 'colspan', 'rowspan', 'draggable'],
+  classNameBindings: [
+    'align',
+    'isGroupColumn:lt-group-column',
+    'isHideable',
+    'isSortable',
+    'isSorted',
+    'column.classNames',
+    'droptargetClass',
+    'dragClass'
+  ],
```

Comment:
You dont need to include all class binding from the Base class since this property get concatenated.